presto-docs/src/main/sphinx/connector/postgresql.rst

```diff
@@ -84,6 +84,106 @@ Finally, you can access the ``clicks`` table in the ``web`` schema::
 If you used a different name for your catalog properties file, use
 that catalog name instead of ``postgresql`` in the above examples.
 
+Type mapping
+------------
+
+PrestoDB and PostgreSQL each support types that the other does not. When reading from or writing to PostgreSQL, Presto converts
+the data types from PostgreSQL to equivalent Presto data types, and from Presto to equivalent PostgreSQL data types.
+
+PostgreSQL type to PrestoDB type mapping
```

Comment:
```suggestion
PostgreSQL to PrestoDB type mapping
```
Change suggested to parallel https://github.com/prestodb/presto/blob/master/presto-docs/src/main/sphinx/connector/mysql.rst